time-tracker-webapps/src/main/webapp/vue-app/components/timeSheet/TimeSheet.vue

```diff
@@ -636,7 +636,7 @@ export default {
       const month = parseInt(isoDate.substring(5, 7));
       const year = parseInt(isoDate.substring(0, 4));
       const startOfMonth = new Date(`${year}-${month}-01 00:00:00Z`);
-      const endOfMonth = new Date(`${year}-${month + 1}-01 00:00:00Z`);
+      const endOfMonth = new Date(`${year}-${(month + 1) % 12}-01 00:00:00Z`);
```

Comment:
we need to increment the year too to get the latest day of the current year with the next instruction

Reply:
argh good catch !